<?xml version="1.0" ?>
<launch>

  <arg name="camera_name"	default="a_phoxi_m_camera"/>
  <arg name="config_dir"	default="$(find aist_localization)/config"/>
  <arg name="camera_frame"	default="$(arg camera_name)_from_b_bot"/>

  <node name="localization" pkg="aist_localization" type="aist_localization"
  	output="screen">
    <param name="camera_frame"	value="$(arg camera_frame)"/>
    <param name="config_dir"	value="$(arg config_dir)"/>
  </node>

</launch>
